### PR TITLE
Date entity + annotation entity refactoring

### PR DESCRIPTION
**Pull request by BasLee**
Visualize the new republic date entity, and implement search button for date entity. 
Besides that, this PR also refactors the project specific parts of the entity annotation rendering. Project specific annotation types and the project specific entity metadata summary component are now part of the project config directory.

### CODE REVIEW

**Comment by BasLee** [comment]
@Doppen will create the new date icon

**Comment by BasLee** [comment]
Implements issue https://github.com/knaw-huc/textannoviz/issues/170

File: src/projects/suriano/config/getEntityCategory.ts
Comment (svandaalen) [inline src/projects/suriano/config/getEntityCategory.ts:1]: Wat is het verschil tussen dit bestand en [src/projects/suriano/annotation/getEntityCategory.ts](https://github.com/knaw-huc/textannoviz/pull/177/files#diff-93a6e88ae5b74666e9a382d0ca13b706c4b0b6be7d7be70703ea825d00e791e9)?

Comment (BasLee) [reply src/projects/suriano/config/getEntityCategory.ts:1]: Gefixt in [59e9542](https://github.com/knaw-huc/textannoviz/pull/177/commits/59e9542e5777798b36356fb12c361bfc5e2859bd)

**Pull request merged**